controllers/datadogagent_controller.go

```diff
@@ -51,6 +51,10 @@ type DatadogAgentReconciler struct {
 // +kubebuilder:rbac:groups=datadoghq.com,resources=datadogagents/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=datadoghq.com,resources=datadogagents/finalizers,verbs=get;list;watch;create;update;patch;delete
 
+// Datadog Metrics
```

Comment:
nit: there's a `Configure External Metrics Server` section below, I think it'd most suited to put it there.

Reply:
Done :+1: